src/emulator/host/include/host/state.h

```diff
@@ -89,3 +89,10 @@ struct HostState {
     GDBState gdb;
 #endif
 };
+
+enum IconSize {
```

Comment:
This is unrelated to the host, it doesn't belong here. Place it in `gui/state.h`

Reply:
oki